kctf-operator/pkg/apis/kctf/v1alpha1/challenge_types.go

```diff
@@ -1,6 +1,6 @@
 // This file is responsible for generating CRD (Custom Resource Definition)
 // kubebuilder might be used to set: default values, optional fields and etc
-// +kubebuilder:validation:Required
+// +kubebuilder:validation:Optional
```

Comment:
does this make everything in this file optional?

Reply:
Yes, it does that, but you can specify some fields that are required. I left this way, because most of the fields are optional.

Reply:
I see, thanks for the clarification. Makes sense